doc/src/build/programming-with-objects/ch5-dynamic-fields.md

```diff
@@ -10,6 +10,14 @@ In previous chapters, we walked through various ways to use object fields to sto
 
 Fortunately, Sui provides *dynamic fields* with arbitrary names (not just identifiers), added and removed on-the-fly (not fixed at publish), which only affect gas when they are accessed, and can store heterogeneous values. This chapter introduces the libraries for interacting with this kind of field.
 
+### Current Limitations
+
+There are some aspects of dynamic fields that are not yet behaving as designed in this early release. We are actively working on these areas, but watch out for:
+
+- `remove` for dynamic fields not giving a full storage refund.
+- The lack of an `exists_` API to check whether a field with a particular name is already defined on an object.
```

Comment:
It exists for `dynamic_object_field`, just not `dynamic_object`. For `dynamic_object`, we only have `exists_with_type`, which returns true iff it has a field with that name and the value has that type.

We probably also want `exists_` which would return true iff it has a field with that name (regardless of the dynamic value type)